tests/misc/test_basic_without_ssh.py

```diff
@@ -75,6 +75,22 @@ def test_checkpoint(self, imported_vm):
         finally:
             snapshot.destroy(verify=True)
 
+    def test_resize_vdi(self, imported_vm):
+        from lib.vdi import VDI
+
+        vm = imported_vm
+        if vm.is_running():
+            vm.shutdown(verify=True, force_if_fails=True)
```

Comment:
Unless I'm mistaken, `imported_vm` is always  supposed to be a stopped VM. It's a global prerequisite in our tests. So let's not attempt to shut it down.

If there currently exists situations where imported_vm can be a running VM (when using the UUID of an existing VM maybe?), then maybe the `imported_vm` fixture should detect that and fail early.

Reply:
It was also an assumed prerequisite that VM to be turned off before calling vdi resize, hence it was added. However since we will handle it within the vdi resize function and in future may offer online vdi resize, I will drop this code.

Reply:
The `class TestBasicNoSSH` uses a `@pytest.mark.usefixtures("started_vm")` hence `imported_vm` is always running in these tests. There is no VM online VDI resize support for now, hence this code will have to stay. @stormi is that ok?

Reply:
Yes, makes sense. However I'm not sure this test belongs to `test_basic_without_ssh` at all. It's not so basic, and it's dependent on the underlying SR. To me it belongs to every SR type test instead (EXT, LVM, NFS, etc.), so that it's tested with each SR type, in addition to linstor.

Reply:
I have removed it from `test_basic_without_ssh` and will add to SR's later.